src/gnatcoll-terminal.adb

```diff
@@ -432,4 +432,24 @@ package body GNATCOLL.Terminal is
          return Internal (Boolean'Pos (Self.FD = Stderr));
       end if;
    end Get_Width;
+
```

Comment:
Please remove the extra blank lines - there should be at most one blank line as separation. (Same goes for lines 452-453.)